src/sentry/integrations/github/client.py

```diff
@@ -375,7 +384,7 @@ def _populate_repositories(self, gh_org: str, cache_seconds: int) -> list[dict[s
             # Remove unnecessary fields from the response
             repositories = [
                 {"full_name": repo["full_name"], "default_branch": repo["default_branch"]}
-                for repo in self.get_repositories(fetch_max_pages=True)
+                for repo in self.get_repos(fetch_max_pages=True)
```

Comment:
Which class is this method defined on? I pulled the branch and am still having trouble finding it personally 😅

Reply:
i also cannot seem to find this method, perhaps we should still have the get_repositories method here but under get_repos?